src/test/rrutil.h

```diff
@@ -176,7 +176,7 @@ static uint64_t GUARD_VALUE = 0xdeadbeeff00dbaad;
  */
 inline static void* allocate_guard(size_t size, char value) {
   char* cp =
-      (char*)malloc(size + 2 * sizeof(GUARD_VALUE)) + sizeof(GUARD_VALUE);
+      (char*)xmalloc(size + 2 * sizeof(GUARD_VALUE)) + sizeof(GUARD_VALUE);
```

Comment:
How do these test changes build, given that tests don't link to `util.cc`? I think you need to declare `xmalloc` in rrutil.h too.